templates/medium/cxf-code-first/medium-fuse-cxf-code-first-blueprint/pom.xml

```diff
@@ -50,27 +44,26 @@
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
-
-  <dependencies>
-    <dependency>
-      <groupId>org.apache.camel</groupId>
-      <artifactId>camel-core</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.camel</groupId>
-      <artifactId>camel-blueprint</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.camel</groupId>
-      <artifactId>camel-cxf</artifactId>
-    </dependency>
-
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-core</artifactId>
+			<version>2.15.1.redhat-621084</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.camel</groupId>
```

Comment:
Why do we need blueprint if only spring/java is supported? and where is camel-spring?